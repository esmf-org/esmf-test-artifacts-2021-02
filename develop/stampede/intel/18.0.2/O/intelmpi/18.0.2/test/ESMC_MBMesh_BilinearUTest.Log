20201107 202737.061 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201107 202737.061 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20201107 202737.061 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20201107 202737.061 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20201107 202737.061 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20201107 202737.061 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20201107 202737.061 INFO             PET0 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_36-22-g504ad1e564
20201107 202737.061 INFO             PET0 ESMF library build date/time: "Nov  7 2020" "18:17:30"
20201107 202737.061 INFO             PET0 ESMF library build location : /home1/07356/tg866552/himanshu/esmf
20201107 202737.061 INFO             PET0 ESMF_COMM                   : intelmpi
20201107 202737.066 INFO             PET0 ESMF_NETCDF                 : enabled
20201107 202737.066 INFO             PET0 ESMF_PNETCDF                : disabled
20201107 202737.066 INFO             PET0 ESMF_PIO                    : enabled
20201107 202737.066 INFO             PET0 ESMF_YAMLCPP                : enabled
20201107 202737.066 INFO             PET0 ESMF_MOAB                   : enabled
20201107 202737.067 INFO             PET0 Beginning Test, file ESMC_MBMesh_BilinearUTest.C, line 71
20201107 202737.067 INFO             PET0 NUMBER_OF_PROCESSORS 1
20201107 202737.069 INFO             PET0 PASS Quadrilateral bilinear weight generation, ESMC_MBMesh_BilinearUTest.C, line 107
20201107 202737.070 INFO             PET0 PASS Spherical quadrilateral bilinear weight generation, ESMC_MBMesh_BilinearUTest.C, line 137
20201107 202737.071 INFO             PET0 PASS Triangle mesh bilinear weight generation, ESMC_MBMesh_BilinearUTest.C, line 167
20201107 202737.071 INFO             PET0 PASS Spherical triangle mesh bilinear weight generation, ESMC_MBMesh_BilinearUTest.C, line 197
20201107 202737.071 INFO             PET0 Ending Test, file ESMC_MBMesh_BilinearUTest.C, line 210
20201107 202737.072 INFO             PET0 Finalizing ESMF
Sat Nov  7 20:48:09 CST 2020
