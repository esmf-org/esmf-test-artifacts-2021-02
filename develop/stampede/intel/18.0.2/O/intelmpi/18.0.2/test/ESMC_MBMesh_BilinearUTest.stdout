TACC:  Starting up job 6717738 
TACC:  Starting parallel tasks... 
Beginning Test, file ESMC_MBMesh_BilinearUTest.C, line 71
NUMBER_OF_PROCESSORS 1
PASS Quadrilateral bilinear weight generation, ESMC_MBMesh_BilinearUTest.C, line 107
PASS Spherical quadrilateral bilinear weight generation, ESMC_MBMesh_BilinearUTest.C, line 137
PASS Triangle mesh bilinear weight generation, ESMC_MBMesh_BilinearUTest.C, line 167
PASS Spherical triangle mesh bilinear weight generation, ESMC_MBMesh_BilinearUTest.C, line 197
Ending Test, file ESMC_MBMesh_BilinearUTest.C, line 210
 PET 0 Test Elapsed Time  138.002000 msec.
TACC:  Shutdown complete. Exiting. 
Thu Oct 29 06:51:25 CDT 2020
